packages/subproviders/test/unit/redundant_rpc_subprovider_test.ts

```diff
@@ -18,6 +20,8 @@ describe('RedundantRpcSubprovider', () => {
         provider = new Web3ProviderEngine();
         const endpoints = ['http://localhost:8545'];
         const redundantSubprovider = new RedundantRPCSubprovider(endpoints);
+        // Hack: Hot-swap rpc with ganacheSubprovider
```

Comment:
What do you think about changing RedundantRPCSuprovider to redundant subprovider that accepts providers/subproviders instead of urls?